ntbs-service/Services/AlertService.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using EFAuditer;
+using ntbs_service.DataAccess;
+using ntbs_service.Models;
+using ntbs_service.Models.Enums;
+
+namespace ntbs_service.Services
+{
+    public interface IAlertService
+    {
+        Task<bool> CreateExampleTbServiceAlertAsync(ExampleTbServiceAlert alert);
+        Task DeleteAlertAsync(int alertId, string userId);
+
+    }
+
+    public class AlertService : IAlertService
+    {
+        private readonly IAlertRepository _alertRepository;
+        private readonly NtbsContext _context;

```

Comment:
We should either be talking to the repo or to the context, otherwise the abstractions get quite muddled. I would suggest service talking to repo, and repo talking to context - see how the notifications classes do it now post Conor's recent changes :)
In this case, it would probably be down to moving the `UpdateDatabase` method over to the repo as `UpdateAlert` or similar